apps/builder/app/shared/shortcuts/shortcuts.ts

```diff
@@ -38,7 +38,7 @@ export const useSharedShortcuts = () => {
     "backspace, delete",
     deleteSelectedInstance,
     // prevent instance deletion while deleting text
-    { enableOnFormTags: false, enableOnContentEditable: false },
+    { enableOnFormTags: true, enableOnContentEditable: false },
```

Comment:
These shortcuts are run for both builder and canvas. And with this option turned on deleting text in builder inputs will delete instance. I guess this need to be split into builder and canvas shortcuts.

Reply:
Ah, right. Instance is deleted when I delete something in a control in style panel.

Reply:
Fixed